spec/serializers/spree_signifyd/credit_card_serializer_spec.rb

```diff
@@ -15,11 +15,11 @@ module SpreeSignifyd
       end
 
       it "expiryMonth" do
-        expect(serialized_credit_card['expiryMonth']).to eq credit_card.month
+        expect(serialized_credit_card['expiryMonth']).to eq credit_card.month.to_i
       end
 
       it "expiryYear" do
-        expect(serialized_credit_card['expiryYear']).to eq credit_card.year
+        expect(serialized_credit_card['expiryYear']).to eq credit_card.year.to_i
```

Comment:
We should always have a year, but `nil.to_i` yields 0, this could probably cause some strange behavior.